examples/backup-local/README.md

```diff
@@ -0,0 +1,17 @@
+# Local GraphDB Backups Example
+
+This folder contains a simple example of how to configure the Helm chart with local GraphDB backups backed by an existing Persistence Volume
+Claim.
+
+## Configuration
+
+The default [values.yaml](values.yaml) in the example uses a simple PVC that should be sufficient for demonstration purposes,
+However, ideally, you should consider using a PVC with cross-zone or cross-region data replication for better resiliency.
+
+Have in mind that the chart does not rotate backups, this is up to you and your retention policy.
```

Comment:
I always find "have in mind" a bit awkward. I think keep / bear in mind are more common.